src/main/java/fi/helsinki/cs/tmc/core/domain/Exercise.java

```diff
@@ -28,10 +28,10 @@
     private String checksum; //: "406f2f0690550c6dea94f319b2b1580c",
 
     @SerializedName("return_url")
-    private String returnUrl; //: "https://tmc.mooc.fi/staging/exercises/284/submissions.json",
+    private String returnUrl; //: "https://example.com/staging/exercises/284/submissions.json",
 
     @SerializedName("zip_url")
-    private String zipUrl; //": "https://tmc.mooc.fi/staging/exercises/284.zip",
+    private String zipUrl; //": "https://example.com/staging/exercises/284.zip",
```

Comment:
I think we could just drop these sample strings here... (no need to do it in this PR but you can if you wish so)